apps/wallet-dashboard/lib/utils/vesting/vesting.spec.ts

```diff
@@ -20,6 +20,8 @@ import {
     getSupplyIncreaseVestingUserType,
 } from './vesting';
 
+const MOCKED_CURRENT_EPOCH_TIMESTAMP = 1733340503570; // Wednesday, 04 December 2024 19:28:23
```

Comment:
It is better that this mocked constant is dynamic
```suggestion
const MOCKED_CURRENT_EPOCH_TIMESTAMP = Date.now() + (MILLISECONDS_PER_HOUR * 6); // 6 hours later
```